.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=weak
   matrix:
   - FRAMEWORK=Codeception SUITES=cli,unit TEST_PATH=. XDEBUG=1 PECL=mongodb
-  - FRAMEWORK=Yii2 TEST_REPO="https://github.com/Codeception/yii2-tests"
+  - FRAMEWORK=Yii2 TEST_REPO="https://github.com/Slamdunk/yii2-tests  -b closeSessionOnRecreateApplication"
```

Comment:
This is the change I was referring to: a PR can be proposed and proved without a separate branch on Codeception/yii2-tests merged